sdk/core/core-rest-pipeline/review/core-rest-pipeline.api.md

```diff
@@ -208,6 +210,7 @@ export interface PipelineRequest {
 export interface PipelineRequestOptions {
     abortSignal?: AbortSignalLike;
     allowInsecureConnection?: boolean;
+    authenticationOptions?: AuthenticationOptions;
```

Comment:
it doesn't seem like this actually gets passed anywhere? You'll need to set it in `ServiceClient` if you want it to be on generated requests.

Reply:
Thanks! I knew I was missing something